src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -66,12 +66,13 @@ <h3>{{ _("Version Signature Pending") }}</h3>
 {% endif %}
 
 <p>
-  {% if addon.type == amo.ADDON_STATICTHEME %}
-    <a class="button" href="{{ url('devhub.themes') }}">
-  {% else %}
-    <a class="button" href="{{ url('devhub.addons') }}">
-  {% endif %}
-    {{ _("Go to My Submissions") }}</a>
+  {% with link_text=_("Go to My Submissions") %}
+    {% if addon.type == amo.ADDON_STATICTHEME %}
+      <a class="button" href="{{ url('devhub.themes') }}">{{ link_text }}</a>
+    {% else %}
+      <a class="button" href="{{ url('devhub.addons') }}">{{ link_text }}</a>
```

Comment:
Tempted to just put the `{% if %}` inside the `href` attribute here... What do you think?

Reply:
yeah.  As it's jinja we could either write out the full `{% if ... }` or use it inline instead, like `{% with link=url('devhub.themes') if addon.type == amo.ADDON_STATICTHEME else url('devhub.addons') %}`

Reply:
https://github.com/mozilla/addons-server/pull/15558/commits/5de6dcaef2e9aed7bb51bb244f6c157ede7d59b9